src/ert/scheduler/driver.py

```diff
@@ -18,6 +18,8 @@ class Driver(ABC):
 
     def __init__(self, **kwargs: Dict[str, str]) -> None:
         self._event_queue: Optional[asyncio.Queue[Event]] = None
+        # we will keep the error messages coming from the driver
+        self._job_error_message: Dict[int, str] = {}
```

Comment:
Maybe the instance variable name can be more descriptive?
```suggestion
        self._job_error_message_by_iens: Dict[int, str] = {}
```